Model/Quote/GetCartLineItems.php

```diff
@@ -73,54 +124,136 @@ public function getItems(CartInterface $quote): array
     private function getLineItem(CartItemInterface $item): array
     {
         $lineItem = [
-            'platform_id' => (string)$item->getProduct()->getId(),
+            'id' => (int)$item->getProduct()->getId(),
             'quantity' => $this->extractLineItemQuantity($item),
+            'title' => $this->getLineItemName($item),
+            'product_title' => $this->getLineItemName($item),
+            'weight' => $this->getLineItemWeightInGrams($item),
+            'taxable' => true, // Doesn't matter since RSA will handle taxes
+            'image' => $this->getLineItemImage($item),
+            'requires_shipping' => $this->getRequiresShipping($item),
             'line_item_key' => (string)$item->getId(),
-            'price_adjustment' => $this->getPriceAdjustment($item),
+            'price' => $this->getLineItemPrice($item),
         ];
+
         $item = $item->getParentItem() ?: $item;
         if ($item->getProductType() === Configurable::TYPE_CODE) {
             $lineItem = $this->addConfigurableOptions($item, $lineItem);
         }
+
+        if ($item->getProductType() === Bundle::TYPE_CODE) {
+            $lineItem = $this->addBundleOptions($item, $lineItem);
+        }
+
         foreach ($this->configuration->getCustomOptions($item) as $customOption) {
             $lineItem = $this->addCustomOptions($customOption, $lineItem);
         }
         return $lineItem;
     }
 
     /**
-     * Get quote item quantity considering product type.
+     * Gets the product's name from the line item
      *
      * @param CartItemInterface $item
-     * @return int
+     * @return void
      */
-    private function extractLineItemQuantity(CartItemInterface $item): int
+    private function getLineItemName(CartItemInterface $item)
     {
-        $parentItem = $item->getParentItem();
-        if ($parentItem) {
-            $item = $parentItem;
+        /** @var CartItemInterface */
+        $item = $item->getParentItem() ?: $item;
+
+        return $item->getName();
+    }
+
+    private function getLineItemPrice(CartItemInterface $item)
+    {
+        /** @var CartItemInterface */
+        $pi = $item->getParentItem();
+        if ($pi) {
+            return static::convertToCents($pi->getPrice());
         }
+        
+        if ($item->getProductType() === Bundle::TYPE_CODE) {
+            $price = 0;
+            foreach ($item->getChildren() as $childItem) {
+                $price += static::convertToCents($childItem->getPrice());
```

Comment:
Is this supports "Fixed Price" bundle products?

Reply:
Fixed it to work for fixed products